lib/fix-me.js

```diff
@@ -35,6 +36,15 @@ FixMe.prototype.run = function(engineConfig) {
   this.find(paths, strings);
 };
 
+var isItsOwnConfig = function(path, lineNumber) {
```

Comment:
wdyt about a different name for this function? 

I find the [test](https://github.com/codeclimate/codeclimate-fixme/pull/47/files#diff-353ae98a4e441f538fae0a4f9df49bccR63) description really informative.

WDYT about something like two functions:

`isItsOwnConfig` && `isNotComment` or something?